tests/InputFiles/ScalarWave/PlaneWave1DObserveExample.yaml

```diff
@@ -18,7 +18,8 @@ AnalyticSolution:
         Phase: 0.0
 
 Evolution:
-  InitialTime: 0.0
+  InitialTime: &InitialTime
```

Comment:
Just so I can understand, what's going on here with &InitialTime and *InitialTime, instead of specifying numbers?

Reply:
the `&InitialTime` means "Create a pointer to this value called InitialTime" and the `*InitialTime` says "Dereference said pointer". Basically, it's a way of only having to specify the initial time in one place but being able to use it in many without manually having to keep them in sync. This is a yaml builtin feature called anchors, I think.